### DISCUSSION TITLE
Unity Spatial_toolkit image different than Spatial My Sandbox

### DISCUSSION THREAD

**Discussion by iweavings** [spatialsys/spatial-unity-sdk#39] [Help]
Hi, 
I'm new to all this. 
I studied a couple of modules on code 2 years ago. passed Game 1. not the second module. (But I finished).
I didn't have the chance to practice what I learned in Unity. But with this last project, (my practice-led final Masters artwork), I hope to create a Virtual Gallery in Spatial, by uploading my own 3D scanned environments. 

I doubt that I will be able to use the 3DGS Model I made of a local Gallery.
I want the scanned architecture, with its natural textures; doesn't look promising so far. (Excuse the soup opera ..)

To the point.
I managed to download the Unity/Spatial tool kit to my computer and configure everything without bugs to start My Sandbox successfully until now.

However, in Unity, I build a climbing box to set up a new staircase for a second floor. That is where I hope to add the scanned 3D Model of the local Gallery.

I understand that we can duplicate, or add code to an object to duplicate it many times. i want to duplicate the already made staircase object from the spatial tool kit create a second floor, and upload the 3D Model of a gallery.

When you look at my Unity picture and compare it with _My Sandbox_, you notice that the object "climbing block" I added there, doesn't appear on My Sandbox. 

Question 
Why would that be? - what would you suggest me to do? 
I thought about deleting my new climbing box, and duplicate my "given staircase.

Will I have issues with the "exporting" my new objects to _My Sandbox_ as it happened here?

Thanks for your attention.
Paula

![CIM500 2 4 Model in Unity Spatial_toolkit](https://github.com/user-attachments/assets/b8b6543f-5ded-43c2-b344-fa4b7d474b39)

![CIM500 2 3 Model in Spatial_MySandbox](https://github.com/user-attachments/assets/e42f8e91-1435-4d8e-ac76-3b8fc32f5557)

**Comment by ProtoArts** [comment]
Is the ladder climbable anyway?
Is the collision Active?

Maybe try to create a prefab with a Cube with a Empty Object child with the climbable and the box collision components.
